### DISCUSSION TITLE
Bedrock & openAI - chat titles come in multiple languages, but i only used english

### DISCUSSION THREAD

**Discussion by dirkpetersen** [danny-avila/LibreChat#4011] [Troubleshooting]
### What happened?

chat titles come in multiple languages, but i only used english. Librechat answers in German and french 

### Steps to Reproduce

I am always asking the same question multiple times in multiple chats while also switching the models in those chats:

"Who is the most famous person in the history of Oregon. Also say which exact name and version of LLM produced the answer"

### What browsers are you seeing the problem on?

Firefox

### Relevant log output

```shell
no log output
```

### Screenshots

![image](https://github.com/user-attachments/assets/e9d71c45-5190-483f-be25-2e0c7c366636)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This can't be reproduced with certainty. Certain tokens will throw the model off with the title output. I have ideas on how to improve the title prompt, but ultimately, the best solution is to allow admins to configure their own title prompt.

See also: https://github.com/danny-avila/LibreChat/discussions/2077
